Web3privacynowplatform/Briefs/Waku.md

```diff
@@ -0,0 +1,85 @@
+**Basic info**
+
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Project name**  | Waku |
+| **Logotype**  | https://github.com/waku-org/docs.waku.org/blob/develop/static/img/logo_white.svg |
+| **Short project description (1 sentence)** | A family of robust, censorship-resistant communication protocols designed to enable privacy-focused messaging for web3 apps. |
+| **Launch date** | March 2022  |
```

Comment:
@fryorcraken So what should we put here? (in context of your last comment)

Reply:
I'd say MVP launch Dec 2023.